index.js

```diff
@@ -1,7 +1,76 @@
 // Iteration 1: Names and Input
-
+const hacker1 = 'jason'
+console.log(`The driver's name is ${hacker1}`)
+const hacker2 = 'Michael'
+console.log(`The navigator's name is ${hacker2}`)
 
 // Iteration 2: Conditionals
-
+if (hacker1.length > hacker2.length) {
+    console.log(`The driver has the longest name, it has ${hacker1.length} characters`)
+} else if (hacker1.length < hacker2.length) {
+    console.log(`It seems that the navigator has the longest name, it has ${hacker2.length} characters`)
+} else {
+    console.log(`Wow, you both have equally long names, ${hacker1.length} characters!`)
+}
 
 // Iteration 3: Loops
+let letterUp = hacker1.toUpperCase();
+let up = '';
+for(let i=0; i<letterUp.length; i++){
+   up += letterUp[i] + ' ';     
+}
+console.log(up)
+
+let reverso = "";
```

Comment:
```suggestion
let reverso = '';
```
Lo mismo por ponerme tiquismiquis, pero vamos, si antes has declarado la variable up con comillas simple, hagamos que todo este igual y asi nuestro TOC interno no explota :D